mysql-test/suite/innodb/t/upgrade_orphan.test

```diff
@@ -148,3 +148,5 @@ let $restart_parameters =;
 --source include/restart_mysqld.inc
 --force-rmdir $MYSQLD_DATADIR1
 --remove_file $MYSQL_TMP_DIR/data57.zip
+--remove_file $MYSQLD_LOG
```

Comment:
Is it upstream, or if it is not upstream, why we are different?

Reply:
This is upstream 8.0.13 issue

Reply:
Ack, please report to them

Reply:
Done at https://bugs.mysql.com/bug.php?id=93686

Reply:
Thanks, please copy to JIRA too